api/desecapi/tests/testdomains.py

```diff
@@ -1,7 +1,7 @@
 from django.urls import reverse
```

Comment:
In `urls.py`, you wrote: 

> Always use `rest_framework.reverse.reverse`, do not directly use `django.urls.reverse`

Should we follow this recommendation here (and in the rest of the code)?

Reply:
yup. Fixed with next push